doc/LLC/LLC_Quickstart_Tutorial.md

```diff
@@ -0,0 +1,181 @@
+
+- Install VS 2019 (Community or higher) and make sure you have the [latest updates](https://www.visualstudio.com/).
+  - Need at least .NET Framework 4.6.1 and 4.7 development tools
+- Install the **.NET Core cross-platform development** workloads in VisualStudio
+- Install **.NET Core 5.0.301 SDK** for your specific platform. (or a higher version within the 5.0.*** band)  (https://dotnet.microsoft.com/download/dotnet-core/5.0)
+- Install the latest version of [git](https://git-scm.com/downloads)
+- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell), version 6 or higher.
+- Install [NodeJS](https://nodejs.org/) (14.x.x).
+
+## Setup your repo
+
+- Fork and clone an [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net) repo. Follow the instructions in the [.NET Contributing.md](https://github.com/Azure/azure-sdk-for-net/issues/12903) to fork and clone the `azure-sdk-for-net` repo.
+- Create a branch to work in. 
+
+## Getting started
+
+For this tutorial, we'll create a getting started project in a branch of your fork of `azure-sdk-for-net` repo. To create the project, we'll copy the [Azure.Template.LLC](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template-LLC) project for .NET.
+
+The `Azure.Template.LLC` project is a great place to get started with Azure SDK .NET library development in general, as it contains a number of common patterns we use for tests, samples, documentation, generating API listings. It also has an example of a generated library for demonstrating LLC APIs.
+
+**Learn more**: see other features listed in the [Azure.Template.LLC README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-LLC/Azure.Template.LLC/README.md).
+
+### Create the project
+
+#### 1. Copy the Azure.Template.LLC directory
+
+Copy the `sdk\template-LLC` directory to an `sdk\<your-service-name>` directory at the same level. For example, from the root of your cloned repo:
+
+```
+% mkdir sdk\<your-service-name>
+% xcopy sdk\template-LLC sdk\<your-service-name> /E
+```
+
+**Learn more:** to understand more about the Azure SDK repo structure, see [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md) in the `azure-sdk` repo.
+
+#### 2. Remove existing generated code
+
+Open the `Azure.Template.LLC.sln` file from the `sdk\<your-service-name>\Azure.Template.LLC` directory.
+
+You'll notice that the `Azure.Template.LLC` project already has generated code in it!  You're welcome to explore what's there -- it's been generated from the swagger file in the `swagger` folder. For this tutorial, however, we'll delete these generated files so we can start clean.
+
+- Delete the Generated folder in the `Azure.Template.LLC` project.
+- Delete the `swagger` folder in the `Azure.Template.LLC` project.
+
+#### 3. Rename the solution and projects
+
+In Visual Studio:
+
+- Rename the `Azure.Template.LLC` project to `Azure.<your-sdk-name>`.
```

Comment:
For continuity with the "Repo Structure" documentation linked above, I think you should explain that `Azure.<your-sdk-name>` is the "package name".  And maybe change all the other instances of `Azure.<your-sdk-name>` to just `<package name>`.

Reply:
Thanks for the suggestion! I'll add the explanation that `Azure.<your-sdk-name>` is the "package name". I think `Azure.` is constant for track-2 libraries, so I added `Azure.<your-sdk-name>`. Please let me know if you still think that we should change `Azure.<your-sdk-name>` to `<package name>`.

Reply:
Thanks for the suggestion! I'll add the explanation that `Azure.<your-sdk-name>` is the "package name. In [this](https://github.com/Azure/azure-sdk-for-net/blob/27af1daac97a674ae723b218a41edbf0287abdbc/doc/LLC/LLC_Quickstart_Tutorial.md#3-rename-the-solution-and-projects) section in Learn more, I've added  [.NET Guidelines on Namespace Naming](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming), where it says

>✅ DO adhere to the following scheme when choosing a namespace: `Azure.<group>.<service>[.<feature>]`

I think `Azure.` is constant for track-2 libraries, so I added `Azure.<your-sdk-name>`. Please let me know if you still think that we should change `Azure.<your-sdk-name>` to something else.

Reply:
As long as there is a clear link to the "Repo Structure" doc I think we are good.  Thanks!